.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: go
+install:
+- go get golang.org/x/tools/cmd/cover
+- go get github.com/mattn/goveralls
+- go get github.com/Masterminds/glide
+- glide install
+script:
+- go test github.com/nikofil/go-dpi
```

Comment:
Should be `mushorg`. I usually check out the repo I have forked and set the remote to my fork in Github. See here: http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html